### PR TITLE
hack around exception/endless recursion, see discussion in #445 

### LINKED ISSUE #445: Recursive update exception
```
[] > trivial
  memory > f1
  cage > f2
  memory > f0
  seq > @
    f0.write 0
    f1.write 1
    f2.write ((f0).add (f1))
    f1.write (f2)
    TRUE
```

Also this way
```
[] > trivial
  cage > f1
  cage > f2
  cage > f0
  seq > @
    f0.write 0
    f1.write 1
    f2.write ((f0).add (f1))
    f1.write (f2)
    f1
```

### PR DESCRIPTION
**Pull request by dours**
maybe closes #445

### CODE REVIEW

File: eo-runtime/src/test/eo/org/eolang/gray/rho-of-add-should-not-change.eo [hunk 1]
```diff
@@ -0,0 +1,15 @@
++package org.eolang
```
Comment (marceloamadeu) [inline eo-runtime/src/test/eo/org/eolang/gray/rho-of-add-should-not-change.eo:1]: @dours see other `EO`, ​​they all have **licenses** at the top. I suggest including this here too.
Tks

File: eo-runtime/src/test/eo/org/eolang/gray/rho-of-add-should-not-change.eo [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
++alias cage org.eolang.gray.cage
++junit
+
+[] > rho-of-add-should-not-change
```
Comment (yegor256) [inline eo-runtime/src/test/eo/org/eolang/gray/rho-of-add-should-not-change.eo:5]: @dours move this test to `runtime-tests.eo`, please

**Comment by yegor256** [comment]
@dours see my comment above

**Comment by dours** [comment]
@yegor256 moved the test

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@dours I think I've made a mistake: a better place for this test would be `cage-tests.eo`. I'll move it, after it's merged.

**Comment by yegor256** [comment]
@dours thanks!

**Pull request merged**